src/message/Message.js

```diff
@@ -153,3 +125,31 @@ ngeo.Message.prototype.getMessageObjects = function(object, opt_type) {
 
   return msgObjects;
 };
+
+
+/**
+ * @enum {string}
+ * @export
+ */
+ngeo.message.Message.MessageType = {
```

Comment:
That's a little bit repetitive for a name...
Can you still change it to `ngeo.message.Message.type` ?
(And the last name should start without uppercase no ?)

Reply:
Should start with a uppercase character. Google does it [this way](https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming).

How about `ngeo.message.Message.Type`?

Reply:
okay

Reply:
Changed it